libs/sdk-ui-dashboard/src/model/store/storeAccessors/SingleDashboardStoreAccessor.ts

```diff
@@ -0,0 +1,68 @@
+// (C) 2022 GoodData Corporation
+
+import { DashboardStoreAccessorRepository } from "./DashboardStoreAccessorRepository";
+import { idRef, serializeObjRef } from "@gooddata/sdk-model";
+import { DashboardDispatch, DashboardSelectorEvaluator, DashboardState } from "../types";
+
+const DASHBOARD = serializeObjRef(idRef("dashboard_id"));
```

Comment:
I think we should use some more distinct string, that cannot clash with normal refs (because a dashboard with id `dashboard_id` will clash with this).